Observatory-Control-System/BLOCKS/block-jira-project-workflow.rst

```diff
@@ -0,0 +1,245 @@
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+.. _BLOCK-Jira-Workflow:
+
+###################
+BLOCK JIRA Workflow
+###################
+
+The Rubin Observatory uses observing blocks to design, plan, execute, and track tasks at the summit which require data taking.
+These tasks range from pure engineering (e.g. closed-dome TMA velocity tests) to science-driven surveys. 
+In most cases, the planning and execution of such tasks must take place on a much shorter timescale than the analysis of the data that is collected, necessitating a separate JIRA project and workflow. 
+This workflow is captured in the BLOCK Project, and described in detail here. 
+
+.. figure:: _static/BLOCK_Jira_workflow.png
+    :name: BLOCK Jira Project Workflow
+    :align: center
+    :width: 480
+
+    A block diagram of the workflow contained in the BLOCK Jira project.
+
+BLOCK Ticket Roles and Workflow
+===============================
+
+BLOCK tickets are heavily relied on during both the task planning and task execution phases, 
+and provide a central point of communication between stakeholders and the summit crew. 
+Since nearly all groups across the observatory are stakeholders in the tests that are carried out at the summit, clear roles and expectations must be defined for how the JIRA BLOCK tickets are used. 
+Here we define the various roles and their expectations, and present a brief overview of the BLOCK Ticket workflow. 
+
+
+Roles and Responsibilities
+--------------------------
+In the JIRA issues, there are only two formal roles defined: Proposer and Assignee. 
+
+**Proposer**
+^^^^^^^^^^^^
+The Proposer is the stakeholder requesting the observing task. 
+They are chiefly responsible for the initial ticket submission, including providing links to supporting materials, and the final ticket sign-off (see Transitioning to Done). 
+They are expected to work closely with the Assignee during the Preparation phase, but are not expected to be involved in planning when a task will take place or during the task execution (unless otherwise required). 
+Anyone can take the role of Proposer as long as they are knowledgeable in the task requirements and completion criteria. 
+
+**Assignee**
+^^^^^^^^^^^^
+The Assignee is the member of the Commissioning Science team expected to oversee the task during the preparation and execution phase. 
+They are expected to collaborate with the Proposer during task preparation, and are directly responsible for ensuring the task is Ready for execution at the summit. 
```

Comment:
I think these two phrases could be in an itemized list. They are important and having them as bullet points help with visibility.

Reply:
Good point. David provided an example formatting below that I will adopt for both Assignee and Proposer sections.